docs/en/snapshotcloneserver_en.md

```diff
@@ -127,7 +127,7 @@ Essencially, the recovery operation is based on cloning. To explain cloning in a
 
 #### 2.3.3 Writing Cloned Volume
 
-The writing process of a cloned volume is just like writing a normal volume. The difference is that the chunk of the cloned volume will record the bitmap. When data is written, the bit in the corresponding position will be set to 1 in order to distinguish the unwritten area that need to be read from the clone source.
+The writing process of a cloned volume is just like writing a normal volume. The difference is that the writing of the cloned volume will record the bitmap. When data is written, the bit in the corresponding position will be set to 1 in order to distinguish the unwritten area that need to be read from the clone source.
```

Comment:
will record the bitmap -》 will be recorded in a bitmap

Reply:
> will record the bitmap -》 will be recorded in a bitmap

done